compliance/compliance.go

```diff
@@ -83,7 +83,7 @@ func SetNonDefaultVRFName(n string) {
 // Test describes a test within the compliance library.
 type Test struct {
 	// Fn is the function to be run for a test. Tests must not error if additional
-	// TestOpt arguments are supplied to them, but silently ignore them so as to allow
+	// TestOpt arguments are supplied to them, but silently ignore them to allow
```

Comment:
"so as to allow" is valid grammar. is there a reason to make this change?

Reply:
reverted